libs/common/src/admin-console/models/response/profile-organization.response.ts

```diff
@@ -51,6 +51,7 @@ export class ProfileOrganizationResponse extends BaseResponse {
   accessSecretsManager: boolean;
   limitCollectionCreation: boolean;
   limitCollectionDeletion: boolean;
+  LimitItemDeletion: boolean;
```

Comment:
Lower camel case for JS/TS:

```suggestion
  limitItemDeletion: boolean;
```

And in all other files. (upper camel case for `getResponseProperty` is correct though.)